### PR TITLE
Fix ESLint 5 deprecation warning

### LINKED ISSUE #48: DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead.
```
(node:36737) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard-react")
(node:36737) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "node_modules/eslint-config-standard-jsx/index.js")
(node:36737) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard")
```

### PR DESCRIPTION
**Pull request by jackytck**
Fix ESLint 5 deprecation warning of experimentalObjectRestSpread. Fixed #48

### CODE REVIEW

**Review by jean-smaug — APPROVED** [review]

**Comment by thecotne** [comment]
fixes #48 (right?)

**Review by webark — APPROVED** [review]

**Comment by thecotne** [comment]
@standard ping

**Review by nstrelow — APPROVED** [review]
LGTM
and why can I approve this?

**Comment by nstrelow** [comment]
@standard @feross Please review and merge this, thanks 😁

**Review by thecotne — APPROVED** [review]

**Review by bandreias — APPROVED** [review]
woop woop, works just great!

**Review by feross — APPROVED** [review]
LGTM!

File: eslintrc.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "parserOptions": {
-    "ecmaVersion": 8,
+    "ecmaVersion": 9,
```
Comment (kopax) [inline eslintrc.json:3]: Hi, we are also having this warning and our `ecmaVersion` is 7, while you upgrade from 8 to 9.

Is this what we also want  upgrade to v9?>

**Pull request merged**